modules/bbp-core/src/test/resources/data/bbp/core/person/v0.1.0/all-fields.json

```diff
@@ -11,6 +11,10 @@
       "@id": "{{base}}/schemas/bbp/core/organization/v0.1.0/5cb48794-1af3-4dcc-9271-540bc82361c2"
     }
   ],
+  "identifier": {
```

Comment:
The current schemaorg context file does not contain **identifier** or **propertyID**